scout/server/blueprints/cases/static/form_scripts.js

```diff
@@ -0,0 +1,96 @@
+var searchTerm = document.getElementById('search_term');
```

Comment:
Moved all the javascript code to a dedicated file, mirroring what we have in other blueprints. It's also best practices.